src/main/java/org/kitteh/irc/client/library/implementation/IRCClient.java

```diff
@@ -189,6 +190,16 @@ public IRCServerInfo getServerInfo() {
         return this.serverInfo;
     }
 
+    @Nonnull
+    @Override
+    public Optional<User> getUser() {
+        final ActorProvider.IRCUser user = this.actorProvider.getUser(this.getNick());
```

Comment:
Add this please for myself:
```// TODO always have a User once we know our details via the server```
because currently a User won't be available until KICL joins a channel and shows up on its WHO output.